explorer/lib/explorer_web/components/labeled_progress_bar.ex

```diff
@@ -0,0 +1,32 @@
+defmodule LabeledProgressBarComponent do
+  use ExplorerWeb, :live_component
+
+  attr :progress, :float, required: true
```

Comment:
Maybe it is too obvious and not needed, but what about a comment documenting that the `progress` is expected to be a number between 0-100?

Reply:
Good idea. Or even better, rename the attribute to `percent_progress` so every call site conveys that fact.